firmware/drivers/citiroc/citiroc.c

```diff
@@ -87,6 +87,27 @@ void citiroc_daq_set_hvps_curr(uint16_t curr)
 	CITIROC->HVPSR |= ((curr & 0xffff)<<16);
 }
 
+void citiroc_calib_set_calibration(uint32_t calibration)
+{
+	uint32_t temp;
+	temp = calibration & 0x7fffffff;
```

Comment:
A better way would be to use masks defined in `citiroc.h` here. But let us see how (if) the implementation will change after I see the VHDL.